sys/dev/nginx/nginx.conf

```diff
@@ -0,0 +1,49 @@
+server {
```

Comment:
not a big fan of having different configurations for dev and prod. for me, they should match 1:1.